audformat/core/utils.py

```diff
@@ -212,6 +212,58 @@ def concat(
         return df
 
 
+def duration(
+        obj: typing.Union[pd.Index, pd.Series, pd.DataFrame],
+        *,
+        num_workers: int = 1,
```

Comment:
I suggest we also add `verbose` and pass it to `to_segmented_index()`.

Reply:
I added the verbose argument.